src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Commands/PriorLearningData/RecognisePriorLearningDataHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MediatR;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.CommitmentsV2.Data;
+using SFA.DAS.CommitmentsV2.Data.Extensions;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.CommitmentsV2.Application.Commands.PriorLearningData
+{
+    public class RecognisePriorLearningDataHandler : AsyncRequestHandler<PriorLearningDataCommand>
```

Comment:
I think the naming here is a bit off. Again I'd expect a verb prefix, in this case maybe `Add` or `Update`. That goes for the command, the handler, and their namespace.